src/test/lib/ApiTest.cpp

```diff
@@ -4227,6 +4227,56 @@ void QuicTest_QUIC_PARAM_CONN_STATISTICS_V2_PLAT(MsQuicRegistration& Registratio
     }
 }
 
+
+void QuicTest_QUIC_PARAM_CONN_ORIG_DEST_CID(MsQuicRegistration& Registration) {
+    // This is the unit test for checking to see if a server has the correct original dest CID.
+    TestScopeLogger LogScope0("QUIC_PARAM_CONN_ORIG_DEST_CID");
+    {
+        MsQuicConnection Connection(Registration);
+        TEST_QUIC_SUCCEEDED(Connection.GetInitStatus());
+        uint32_t SizeOfBuffer = 8; // 8 bytes is the expected minimum size of the CID.
+        TestScopeLogger LogScope1("GetParam test success case");
+        QUIC_BUFFER buffer = {SizeOfBuffer, new uint8_t[SizeOfBuffer]};
```

Comment:
You can't use `new` in kernel space. The test is compiled for both user and kernel space. I don't think `new` is really needed. Just allocate it on the stack.

Reply:
See the failed github build jobs.

![image](https://github.com/microsoft/msquic/assets/6022514/f1009d94-0eaa-4d06-97e2-91e0e7982f05)

Reply:
Wow! I learned something new today. You can't use the directly while in Kernel mode. Good catch!